srcpkgs/cherrytree/template

```diff
@@ -1,25 +1,14 @@
 # Template file for 'cherrytree'
 pkgname=cherrytree
-version=0.38.9
+version=0.99.21
 revision=1
-archs=noarch
-build_style=python2-module
-pycompile_dirs="usr/share/cherrytree/modules"
-hostmakedepends="gettext python-devel desktop-file-utils"
-depends="pygtksourceview python-dbus python-enchant desktop-file-utils"
+wrksrc=${pkgname}_${version}
+build_style=cmake
+hostmakedepends="cpputest uchardet-devel libcurl-devel sqlite-devel libxml++-devel gtksourceviewmm-devel gspell-devel gtkmm-devel gettext pkg-config desktop-file-utils"
```

Comment:
Break this line at around 80 columns or so.

Most of these dependencies (the `devel` ones) should go in `makedepends`.